src/noun.substance

```diff
@@ -1488,6 +1502,7 @@ w: magnesium
 d: a light silver-white ductile bivalent metallic element; in pure form it burns with brilliant white flame; occurs naturally only in combination (as in magnesite and dolomite and carnallite and spinel and olivine)
 hs: bitter_spar
 hs: carnallite
+hs: dolomite[1]
```

Comment:
double check with @odanoburu if that is the right way to add sense key.

Reply:
yep, that's the new syntax! `mill-mode` should be highlighting it properly too

Reply:
Yes, it is highlighting it properly